deliverables/homework.playground/Contents.swift

```diff
@@ -1,18 +1,131 @@
-//: Shopping List - Week 1
-
 import Foundation
 
-//: Create an empty array to store shopping list items
+//Walking through homework assignments step-by-step
+
+
+// Question #1) 
+//=============
+// Create an empty array to store shopping list items
+
+var shoppingList: [String] = []
+
+
+        // Other examples that work
+
+        var shoppingList2 = [String]()
+        var shoppingList3 = Array<String>()
+
+
+
+// Question #2)
+//=============
+// Add at least 5 items to the array using 3 different insertion methods
+
+shoppingList = ["apples"]
+shoppingList.append("bananas")
+shoppingList.insert("bread", atIndex: 2)
+
+        // Other examples that work for adding items to an array
+
+shoppingList += ["milk"]
+shoppingList.appendContentsOf(["candy"])
+
+shoppingList2 = ["eggs", "butter"]
+shoppingList.insertContentsOf(shoppingList2, at: 0)
+
+print(shoppingList)
+
+
+
+
+// Question #3)
+//=============
+// Sort your array of items
+
+shoppingList.sort()
+
+
+
+// Question #4)
+//=============
+// Create a function to print an array, with it's index, with one item per line
+
+
+func printArray(array: [String]) {
+    
+    var count: Int = 0
+    for item in array {
+        print ("\(count) - \(item)")
+        count = count + 1
+    }
```

Comment:
As you may have seen in the homework answer, you can use array.enumerate() instead of incrementing a count:
```swift
for (index, item) in array.enumerate() {
    print("\(index) - \(item)")
}
```